### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by brytebee**
### 5. Create these sections of the website for the 3 screen sizes

### CODE REVIEW

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello Bright Atsighi & Collins Tatang** 
Amazing job :tada: :tada: :tada: 
Now you are ready to continue with your next challenge :muscle: 

#### Feedback

- [ ] Don't forget to remove commented code from your index.html file
 
Keep going like this  :clap: :clap:
 
Oscar Andrés Russi Porras

File: style.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  font-size: 0.9rem;
+}
+
+.font-size-smallest {
+  font-size: 0.95rem;
+}
+
+.category {
+  display: block;
+  position: absolute;
```
Comment (Spayco) [inline style.css:85]: - [ ] Use `d-block position-absolute` Bootstrap classes.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  border-bottom: none;
+  color: #fff;
+}
+
+.card-1 {
+  padding: 1rem;
+}
+
+.vs-debate {
+  position: absolute;
```
Comment (Spayco) [inline style.css:156]: - [ ] Use `position-absolute` Bootstrap class.

**Review by Spayco — APPROVED** [review]
# Project Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone. but keep in mind **all these changes unless marked optional will be MANDATORY in the third milestone**.

- [ ] For Desktop size, `LATEST NEWS` should come after `MORE STORIES`
- [ ] For Desktop size, there's some extra `SPONSORED INSIGHT` on the bottom
- [ ] Turn the articles into links.


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 

## However, there are some changes required 👀 🧰 🔨 🔧 
- The top navbar of your page does not look like the original. Please center the Newsweek logo. Also, the newsweek logo is supposed to be a link.

| Your version | Original version |
| --- | --- |
| image | image |

- Also, please note that in the previous image, the VS badge in the 'THE DEBATE' section is misplaced. Please make sure it looks like the original.

- Your Sponsored Insight and Subscription sections are very different from the original. Please fix this:

| Your version | Original version |
| --- | --- |
| image | image |

- Your In The Magazine section looks similar to the original, but notice how the first article has the same height as the remaining articles to it's right. Please fix this:

| Your version | Original version | 
| --- | --- |
| image | image |

- Your version is missing the U.S., World, Business, etc sections. Please add these before requesting for another review:

image

- The Subscribe and Newsletter sections are supposed to have the same height. Also, the footer of the page is supposed to have the same width as the main column. Please fix this:

| Your version | Original version |
| --- | --- |
| image | image |

- Finally, on mobile and tablet views, your page is overflowing to the right. This makes it possible for the user to scroll to the right, which creates a gap on the right side of the page.

image


### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] the login button in the navbar would change to an icon on the mobile screen
- [ ] the subscribe and newsletter sections should have the same width on the desktop size

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by uryelah — APPROVED** [review]
Hi **Bright Atsighi & Collins Tatang**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job implementing all the requests from the previous review!

You did a great job working out the responsiveness of the project and I specially like how you even used pseudo elements in your CSS!

Good luck with the next milestone! 🚀  

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---
    
    
**Keep being an awesome dev and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**